column_buffer.go

```diff
@@ -829,7 +830,11 @@ func (col *booleanColumnBuffer) WriteBooleans(values []bool) (int, error) {
 
 func (col *booleanColumnBuffer) WriteValues(values []Value) (int, error) {
 	var model Value
-	col.writeValues(makeArrayValue(values, unsafe.Offsetof(model.u64)), columnLevels{})
+        if cpu.IsBigEndian {
+                col.writeValues(makeArrayValue(values, unsafe.Offsetof(model.u64)+7), columnLevels{})
```

Comment:
Could you share some context on why a 7 byte offset is needed here?

Reply:
On a big endian system, a boolean or a byte value, which is in little endian format, is byte aligned to the 7th byte in a u64 (8 bytes) variable.. This is the lowest byte on a big endian system but the same will become 7th byte when interpreted to a little endian byte format. So, in order to access 1 byte out of u64 variable, we have to have an offset of "7"...